### PR TITLE
component: Signnupad PR

### PR DESCRIPTION
**Pull request by karenheyn**
completed sign up ad component as per issue #100 
the component accepts the props of heading and subheading 
the sub-components are button-pink and form-small

### CODE REVIEW

File: client/src/components/BookTripCard/BookTripCard.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.Card img {
+  width: 45%;
+  height: 100%;
```
Comment (erinkelley27) [inline client/src/components/BookTripCard/BookTripCard.css]: You may need to change from percentages to rem. Check in with the responsive team about all percentages used in your CSS.

File: client/src/components/SignUpAd/SignUpAd.stories.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import SignUpAd from "./SignUpAd";
+import { tsPropertySignature } from "@babel/types";
+
+const handleClick = e => console.log("clicked");
```
Comment (erinkelley27) [inline client/src/components/SignUpAd/SignUpAd.stories.js]: Add a preventDefault() method here.

File: client/src/components/BookTripCard/BookTripCard.css (hunk 1 above)
Comment (karenheyn) [reply client/src/components/BookTripCard/BookTripCard.css]: this is for a different component on a seperate branch...

File: client/src/components/MemberProfilePage/MemberProfilePage.css [hunk 3]
```diff
@@ -0,0 +1,11 @@
+.MemberProfilePage {
```
Comment (erinkelley27) [inline client/src/components/MemberProfilePage/MemberProfilePage.css]: Is this part of the Sign Up Ad component?

**Pull request merged**